### ISSUE TITLE
Drawing view/ camera clip end and object visibility

### ISSUE THREAD

**Issue by myoualid** [IfcOpenShell/IfcOpenShell#3702]
Currently, the camera clip_end defaults to 10m for all drawing views created.  This  has an incidence on what objects are shown/hidden in a drawing view. 

I'm sure this is in the plans.... but just in case:

- [ ] Can this info be saved in the IFC,
- [ ] When this moves to the drawings tab, can we have the option to choose the height of the drawing cut ( along with the view extents?)

**Comment by Moult** [comment]
The clip end is already saved into IFC. And yes part of the plan is to have a dedicated, tailored UI for this stuff.

**Comment by myoualid** [comment]
@Moult just tested it again - the clip end doesn't save. It's a quick fix but I'm refraining from touching on these modules as I know it's in the plans

**Comment by theoryshaw** [comment]
I think you have to `bim.create_drawing` for it to register the camera bounds in the IFC file.

**Comment by Moult** [comment]
Yeah it's a known issue that fresh cameras don't save. This'll be solved as part of the new UI.

**Comment by sboddy** [comment]
Old issue. I believe this is resolved with the Depth field in the Drawing tab.
